Sources/Testing/Running/Runner.Plan.swift

```diff
@@ -133,11 +134,11 @@ extension Runner.Plan {
   /// - Returns: Whether or not the specified test is selected. If
   ///   `selectedTests` is `nil`, `test` is considered selected if it is not
   ///   hidden.
-  private static func _isTestIncluded(_ test: Test, in selectedTests: Test.ID.Selection?) -> Bool {
-    guard let selectedTests else {
+  private static func _isTestIncluded(_ test: Test, using predicate: Configuration.TestFilter?) -> Bool {
```

Comment:
Let's update the param name to `filter` to match the renamed typealias

Reply:
Don't forget the documentation comment.

Reply:
Done!